keps/sig-node/2400-node-swap/README.md

```diff
@@ -117,6 +118,19 @@ support to nodes in a controlled, predictable manner so that Kubernetes users
 can perform testing and provide data to continue building cluster capabilities
 on top of swap.
 
+Since swap enablement is crucial to many different parties, this KEP aims to
+introduce basic swap enablement and leave further extensions to follow-up KEPs,
+that are already being worked on and will be introduced in the very near future.
+This way Kubernetes users / vendors would be able to use swap in a basic manner
+quickly while extensions would be brought to discussion in dedicated KEPs that
+would progress in the meantime.
+
+For example, to achieve this goal, this KEP does not introduce any new APIs.
+Instead, the behaviour is automatic and implicit that requires minimum user
+intervention (see [proposal below](#steps-to-calculate-swap-limit) for more details).
+As mentioned above, in the very near future, follow-up KEPs would bring API extension
+and customizability, supporting zram/zswap, and many other extensions to discussion.
```

Comment:
I tested this feature with zram on fedora 40 and I didn’t see any issues. I think zswap may need some work.

Reply:
Thanks! Removed zram, left zswap in there.